docs/conf.py

```diff
@@ -253,6 +253,39 @@
     'only_warn_on_example_error': True,
 }
 
+# -- Linking to OpenCV docs by using rst_epilog --------------------------------
+try:
+    import requests
+    from bs4 import BeautifulSoup
+
+    base_url = "https://docs.opencv.org"
+
+    # The stable-version docs are the first item in the second list on the main page
+    all_docs = BeautifulSoup(requests.get(base_url).text, 'html.parser')
+    version = all_docs.find_all('ul')[1].li.a.attrs['href'][2:]  # strip leading "./"
+
+    # Find the relative URL to the page for the `cv` namespace
+    stable_docs = BeautifulSoup(requests.get(f"{base_url}/{version}/namespaces.html").text,
+                                'html.parser')
+    cv_namespace = stable_docs.find("a", string="cv").attrs['href']
+
+    # Find the relative URL for warpAffine in the `cv` namespace
+    all_cv = BeautifulSoup(requests.get(f"{base_url}/{version}/{cv_namespace}").text,
+                           'html.parser')
+    warpAffine = all_cv.find("a", string="warpAffine").attrs['href'][6:]  # strip leading "../../"
+
+    # Construct the full URL for warpAffine
+    warpAffine_full = f"{base_url}/{version}/{warpAffine}"
+except:
+    # In the event of any failure (e.g., no network connectivity)
+    warpAffine_full = ""
+
+rst_epilog = f"""
```

Comment:
Putting this at the end of *every* .rst file seems a bit much - I presume there's not an easy way to put this just on the pages that it's needed?

Reply:
It's not particularly an issue putting it everywhere, and putting it everywhere means that you don't have to do anything to use it.

Reply:
A different approach would be to programmatically add it to the specific docstring where it is needed, but that would be fiddly if we ever rename the function or want to use the shortcut in more places.  If this showed up in the rendered pages, it'd of course be untenable, but it's essentially invisible to the reader.